build.gradle

```diff
@@ -705,6 +705,8 @@ uiTestTargets.each { version ->
     systemProperty 'ui-test.intellij.version', version
     systemProperty 'ui-test.plugin.path', buildPlugin.outputs.files[0].path
 
+    outputs.upToDateWhen { false }
```

Comment:
This is pretty useful... so that the UI tests are always re-run (as that's what you'd rather expect, at least locally)